modules/secure-serverless-harness/versions.tf

```diff
@@ -37,10 +37,10 @@ terraform {
   }
 
   provider_meta "google" {
-    module_name = "blueprints/terraform/terraform-google-cloud-run:secure-cloud-harness/v0.6.0"
+    module_name = "blueprints/terraform/terraform-google-cloud-run:secure-cloud-harness/v0.5.0"
   }
 
   provider_meta "google-beta" {
-    module_name = "blueprints/terraform/terraform-google-cloud-run:secure-cloud-harness/v0.6.0"
+    module_name = "blueprints/terraform/terraform-google-cloud-run:secure-cloud-harness/v0.5.0"
   }
```

Comment:
We should let `release-please` increment the version type as needed
but, since this module name is being changed, we do need to update the name itself
@bharathkkb any thoughts here w.r.t. changing submodule names/continuity?